src/Mono.Android/Mono.Android.targets

```diff
@@ -177,7 +177,7 @@
       Inputs="$(TargetPath);@(ApiCompatibilityFiles)"
       Outputs="$(IntermediateOutputPath)CheckApiCompatibility.stamp">
     <CheckApiCompatibility
-        ApiCompatPath="$(XamarinAndroidSourcePath)\packages\microsoft.dotnet.apicompat\5.0.0-beta.20078.1\tools\net472\"
+        ApiCompatPath="$(NuGet_Packages)\microsoft.dotnet.apicompat\5.0.0-beta.20078.1\tools\net472\"
```

Comment:
I think this change will only work when the `NUGET_PACKAGES`  variable is explicitly set in your environment? I don't believe that this is set by NuGet by default, so this is likely a breaking change.